css/intro.css

```diff
@@ -88,7 +88,7 @@
 
 @media screen and (max-width: 500px) {
   .input-big {
-    height: 42rem;
+    height: 40rem;
```

Comment:
can you please add a screenshot of how this looks?

Reply:
https://user-images.githubusercontent.com/107163260/231949128-f4d7780d-298d-4565-ac60-f29f8fd8562b.mp4

Reply:
Thanks for the screenshots. 
@shubhamsigdar1 I am pretty sure you need not use the `height` property in input bg to set a height for the paragraph as it will automatically be set to the maximum height of the content by the browser. 

Is there any reason we are using this height property for `introduction` paragraph ?

Reply:
I am using join.css If you check this link
https://github.com/Real-Dev-Squad/website-www/blob/develop/css/join.css#L75

Reply:
@yesyash when i removed the height in join.css then now height property in input bg is automatically set to the maximum height of the content by the browser.

Reply:
screenshot of Intro page after removing height from join.css then now height property in input bg is automatically set to the maximum height of the content by the browser in intro page.
<img width="752" alt="image" src="https://user-images.githubusercontent.com/107163260/231960926-0d6f789c-122e-46cf-aa11-7edf080907d1.png">

I am thinking to change classname `input big` to something else so that join.css...css not apply to intro.css...Is I am right @yesyash??

Reply:
Sorry for late reply @yesyash 
Yes after removing className there is **no** unwanted behaviour going on. 
<img width="462" alt="image" src="https://user-images.githubusercontent.com/107163260/232355343-74145292-57ab-48e1-9c64-c1a1473a5f91.png">

Reply:
Great thanks for confirming.